src/include/schema.h

```diff
@@ -121,6 +121,23 @@ struct __wt_table {
 	    &S2C(session)->checkpoint_lock, WT_SESSION_LOCKED_CHECKPOINT, op)
 
 /*
+ * WT_WITH_CHECKPOINT_LOCK_NOWAIT --
+ *	Return error if unable to acquire the checkpoint lock,
+ *	perform an operation if lock acquired, drop the lock.
+ */
+#define	WT_WITH_CHECKPOINT_LOCK_NOWAIT(session, ret, op) do {		\
```

Comment:
@sueloverso : The description for drop seems to be outdated, and I will work on updating it with this change.
I am inclined to follow drop and keep bulk_cursor's checkpoint_wait parameter undocumented.